### DISCUSSION TITLE
Features request

### DISCUSSION THREAD

**Discussion by DanRotaru** [File-New-Project/EarTrumpet#935] [Ideas]
1. Add possibility to turn off this animation

![img](https://i.ibb.co/64qdsBb/image.png)

2. Maybe caused by 1 or not, but when I'm opening earTrumpet I have **lags** ([Video](https://skr.sh/v/151021/1isLVUPv.mp4))

I have Amd Ryzen 7 4700U, and it's not about my hardware :)

Waiting updates to fix that. Thank you!

**Comment by riverar** [comment]
Thanks for the request. Just a heads up, #2 is a dupe of #391
